Tests/eZ/Publish/FieldType/TweetTest.php

```diff
@@ -0,0 +1,199 @@
+<?php
```

Comment:
The potential problem I see with this approach would be that if you create a diff between tags, it would also show these optional files. But generally, I agree that tests should go right after commits that add tested functionality.